.github/workflows/test.yaml

```diff
@@ -91,19 +94,21 @@ jobs:
 
   macos:
     runs-on: macos-12
+    env:
+      MESON_BUILD_OPTS: -Dclosures=0
     steps:
       # Checkout repo
       - uses: actions/checkout@v2
       - name: Install dependencies
         run: |
           brew update
-          for pkg in autoconf automake coreutils gettext libffi libtasn1 libtool; do
+          for pkg in autoconf automake coreutils gettext libffi libtasn1 libtool meson; do
             brew install $pkg || true
```

Comment:
Any reason you don't just use `brew install coreutils gettext libffi libtasn1 meson` instead of the loop?